integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/CatalogsPageTest.java

```diff
@@ -0,0 +1,147 @@
+ */
+
+package com.datastrato.gravitino.integration.test.web.ui;
+
+import com.datastrato.gravitino.integration.test.container.ContainerSuite;
+import com.datastrato.gravitino.integration.test.container.HiveContainer;
+import com.datastrato.gravitino.integration.test.web.ui.Pages.CatalogsPage;
+import com.datastrato.gravitino.integration.test.web.ui.Pages.MetalakePage;
+import com.datastrato.gravitino.integration.test.web.ui.utils.AbstractWebIT;
+import org.apache.hadoop.hive.conf.HiveConf;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+@Tag("gravitino-docker-it")
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class CatalogsPageTest extends AbstractWebIT {
+  private static final ContainerSuite containerSuite = ContainerSuite.getInstance();
+  MetalakePage metalakePage = new MetalakePage();
+  CatalogsPage catalogsPage = new CatalogsPage();
+
+  private static final String metalakeName = "metalake_name";
+  String catalogName = "catalog_name";
+  String modifiedCatalogName = catalogName + "_edited";
+  String schemaName = "default";
+  String tableName = "employee";
+
+  @BeforeAll
+  public static void before() {
+    containerSuite.startHiveContainer();
+
+    // Initial hive client
+    HiveConf hiveConf = new HiveConf();
+    String hiveMetastoreUris =
+        String.format(
+            "thrift://%s:%d",
+            containerSuite.getHiveContainer().getContainerIpAddress(),
+            HiveContainer.HIVE_METASTORE_PORT);
+    hiveConf.set(HiveConf.ConfVars.METASTOREURIS.varname, hiveMetastoreUris);
+  }
+
+  @AfterAll
+  public static void after() {
+    try {
+      closer.close();
+    } catch (Exception e) {
+      LOG.error(e.getMessage(), e);
+    }
+  }
+
+  @Test
+  @Order(1)
+  public void testCreateHiveCatalog() {
+    // Create metalake first
+    metalakePage.createMetalakeBtn.click();
+    metalakePage.setMetalakeNameField(metalakeName);
+    metalakePage.submitHandleMetalakeBtn.click();
+    metalakePage.clickMetalakeLink(metalakeName);
+    // Create catalog
+    catalogsPage.createCatalogBtn.click();
+    catalogsPage.setCatalogNameField(catalogName);
+    catalogsPage.setCatalogCommentField("catalog comment");
+    String hiveMetastoreUris =
+        String.format(
+            "thrift://%s:%d",
+            containerSuite.getHiveContainer().getContainerIpAddress(),
+            HiveContainer.HIVE_METASTORE_PORT);
+    catalogsPage.setHiveCatalogURI(hiveMetastoreUris);
+    catalogsPage.addCatalogPropsBtn.click();
+    catalogsPage.setCatalogProps(1, "key1", "value1");
+    catalogsPage.addCatalogPropsBtn.click();
+    catalogsPage.setCatalogProps(2, "key2", "value2");
+    catalogsPage.handleSubmitCatalogBtn.click();
```

Comment:
Where to select catalog type?
Don't need select `Hive` catalog type?

Reply:
The default provider is hive, so there is no need to select the provider.
The selection of provider can be added to include other catalogs for testing.